ocfweb/stats/printing.py

```diff
@@ -65,10 +66,67 @@ def _semester_histogram():
 
 @periodic(3600)
 def _toner_changes():
-    return [
-        (printer, _toner_changes_for_printer(printer))
-        for printer in ALL_PRINTERS
-    ]
+    return [(printer, _toner_used_by_printer(printer)) for printer in ACTIVE_PRINTERS]
+
+
+# Toner numbers can be significantly noisy, including significant diffs
+# whenever toner gets taken out and put back in whenever there is jam. Because
+# of this it's hard to determine if a new toner is inserted into a printer to
+# reduce this noise we only count diffs that are smaller than a cutoff which
+# empirically seems to be more accurate
+def _toner_used_by_printer(printer, cutoff=.05, since=date(2017, 8, 20)):
```

Comment:
can you put this comment in a docstring so that it will show up in pydoc? (some IDEs and similar can also show this i think)

e.g.

```python
def _toner_used_by_printer(...):
    """Return toner changes for a printer.

    Toner numbers can be significantly noisy...
    """
```